src/mosaicfit.cc

```diff
@@ -801,6 +801,9 @@ void KDTree::printSource() const {
 }
 
 KDTree::Ptr lsst::meas::mosaic::kdtreeMat(SourceMatchGroup &matchList) {
+    if (matchList.size() == 0) {
+        raise std::runtime_error("Can't create kd-tree for empty match list");
```

Comment:
Doesn't C++ use `throw`? (I'm surprised if this compiles.)
Any reason not to use an LSST exception like `lsst::pex::exceptions::RuntimeError`?

Reply:
I was gonna put it here 
https://github.com/lsst/meas_mosaic/commit/4bdc196d69919af5864bcb6e67ce9510cdce7a93

Reply:
You should do both.

Reply:
Looks like you’re suspicion was well-founded:
https://ci.lsst.codes/blue/organizations/jenkins/stack-os-matrix/detail/stack-os-matrix/28863/pipeline

Reply:
Sorry; I blew it.  Syntax highlighting should have told me.

Reply:
I agree that checking in both places is good.